app/views/admin/users/_user.html.erb

```diff
@@ -23,6 +23,9 @@
       <ul class="dropdown-menu dropdown-menu-right" role="menu">
         <li><%= link_to t(:discussions, default: 'Discussions'), admin_user_path(user), remote: true %></li>
         <li><%= link_to t(:edit_user, default: 'Edit User'), admin_user_path(user, mode: 'edit'), remote: true %></li>
+        <li><%= link_to t(:delete_user, default: 'Delete User'), admin_user_path(user), remote: true,
+                        method: :delete,
+                        data: { confirm: "Are you sure you want to delete #{user.name}?" } if current_user.is_admin? %></li>
```

Comment:
As I mentioned above, I think that instead of deleting immediately, we need some kind of intermediate view/modal/warning that says something to the effect of "This user has created X number of tickets.  Do you want to keep these?" 

If the answer is yes, then the user_id should be switched to 2 which is the system user.  An internal note should be added to the history of any topics documenting this change.